docs/ROADMAP.md

```diff
@@ -0,0 +1,442 @@
+PRESC Project Roadmap
+=====================
+
+This is an overview of the features planned for integration into PRESC.
+It is intended to give a high-level description of how these will work and
+sample use cases.
+Prioritization and implementation details are maintained in the repo
+[issues](https://github.com/mozilla/PRESC/issues).
+
+At the core of PRESC is a collection of analyses that can be run on a given
+statistical model and dataset to inform the developer on different aspects of
```

Comment:
nit: "Or combination of model and dataset".

Phrasing could be a bit more direct here. This is the one sentence that contextualizes all that comes afterwards in terms of what the goals of the PRESC project are.